packages/playground/blueprints/src/lib/steps/run-sql-file.ts

```diff
@@ -0,0 +1,48 @@
+import { StepHandler } from '.';
+
+/**
+ * @inheritDoc runSqlFile
+ * @hasRunnableExample
+ * @example
+ *
+ * <code>
+ * {
+ * 	    "step": "runSqlFile",
+ * 		"path": "wordpress.org/plugins"
+ * }
+ * </code>
+ */
+export interface RunSqlFileStep {
+	/**
+	 * The step identifier.
+	 */
+	step: 'runSqlFile';
+	/**
+	 * The SQL file to run.
+	 */
+	path: string;
+}
+
+/**
+ * Run an SQL file.
+ *
+ * @param playground The playground client.
+ * @param path The path to the SQL file.
+ */
+export const runSqlFile: StepHandler<RunSqlFileStep> = async (
+	playground,
+	{ path },
+	progress?
+) => {
+	progress?.tracker.setCaption(`Executing SQL File`);
+
+	const runPhp = await playground.run({
+		code: `<?php
+		require_once '/wordpress/wp-load.php';
```

Comment:
Nit: Let's use `php.documentRoot` instead of hardcoding `/wordpress`. For example `wp-now` may use the actual path on the disk that will be different than `/wordpress`

Reply:
@adamziel This file has been removed.

https://github.com/WordPress/wordpress-playground/pull/745/files